docker/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to the fyne-cross docker images will be documented in this f
 Release cycle won't follow the fyne-cross one, so the images will be tagged using the label
 year.month.day along with the latest one.
 
+# Release 20.12.13
+- Update Go to v1.14.13
+- Fix fails to build for linux mobile #19
```

Comment:
Could this perhaps be “Fix build failure for Linux mobile” instead?

Reply:
Thanks for the catch, fixed in b36aee34dc25a7404bef1d0edd6c61efceb257fe